src/Bundle/Command/DebugResourceCommand.php

```diff
@@ -17,8 +17,8 @@
 use Sylius\Component\Resource\Metadata\Operation;
 use Sylius\Component\Resource\Metadata\Operations;
 use Sylius\Component\Resource\Metadata\RegistryInterface;
-use Sylius\Component\Resource\Metadata\Resource as ResourceMetadata;
 use Sylius\Component\Resource\Metadata\Resource\Factory\ResourceMetadataCollectionFactoryInterface;
+use Sylius\Component\Resource\Metadata\ResourceMetadata;
```

Comment:
On separate note, we have now

> Resource\Metadata\Metadata
> Resource\Metadata\MetadataResource
> Resource\Metadata\Resource\ResourceMetadataCollection

and where `Metadata` and `ResourceMetadata` are not related at quick glance.

What do you think renaming `ResourceMetadata` to `SyliusResource` since this is end-user facing class and it can be branded?

Reply:
I see it's resolved `AsResource` in separate PR
https://github.com/Sylius/SyliusResourceBundle/pull/757